addons/web_editor/static/src/components/media_dialog/file_selector.js

```diff
@@ -313,13 +342,60 @@ export class FileSelector extends Component {
     }
 
     /**
-     * This is used (debounced) to be called after loading an attachment.
-     * This way, the user can always see the "load more" button.
+     * Triggers a small scroll to update the bottom bar: it will trigger the
+     * scoll event which will be handled by the onScroll function below.
+     */
+    triggerScroll() {
+        if (this.allAttachmentsRef.el) {
```

Comment:
Because `triggerScroll` is called by `fetchAttachments` (either directly or via `this.debouncedScroll()`) and this function is called in the `onWillStart`, which happens before the component is mounted in the DOM, which means that in that case, the element doesn't exist (yet) 
In the other cases, it doesn't need "protection" because everything happens after the component has been mounted so the element always exists :slightly_smiling_face: